### ISSUE TITLE
CNTK_204 working properly?

### ISSUE THREAD

**Issue by tomasek** [Microsoft/CNTK#2736]
Hello,

i have been working with this tutorial for sometime. The datasets i have been using are different from the ones provided. 
While training, after around 40 epochs, it reaches stable error and loss values (12.8% and 0.9, respectively).
But, once i try evaluation, it always fails totally the predictions. i tried the python code and adapted the C# evaluation code. 
Even when trying the same sentences on the training dataset on the evaluation, it fails totally. Not even one output in the sequence is correct.

How is it possible that he error rate while training is so low and then fails so much afterwards?

**Comment by hellosaumil** [comment]
@tomasek 
Yeah, I tried to run `CNTK_204_Sequence_To_Sequence.ipynb` under `Tutorials` and it ran successfully on CPU.

What is your python version, cntk version? Are you using CPU/GPU?

**Comment by tomasek** [comment]
@hellosaumil 
it runs perfectly for me too. But after, once i go evaluate, the results are really poor.
I ran it on Azure, so, python 3.5, cntk 2.3 and k80 GPU.
